commandlineparser/include/Parser.h

```diff
@@ -34,7 +34,7 @@ namespace cmdParser
 
 		//parses_the_input data and gives key value pair in  string type.
 		bool Parse(int argc,char* argv[]);
-
+		StorgeType ValueAsString;
```

Comment:
why public ?  comment before the variables whats it is used for

Reply:
corrected.